debian/rules

```diff
@@ -32,6 +32,10 @@ override_dh_auto_build:
 	ninja -C build
 
 override_dh_auto_test:
+	tar xzf ./packaging/trainset.tar.gz -C build
+	tar xzf ./packaging/valset.tar.gz -C build
+	tar xzf ./packaging/testset.tar.gz -C build
+	cp ./packaging/label.dat build
```

Comment:
To avoid license complication and bloated git repo, you can create a "dataset" git repo and package its files in .deb and .rpm and refer to it at build-time (BuildRequires).

Reply:
Anyway, it's fine to proceed with this and move files to another git repo later.

Reply:
Yes. this is generated by the nntrainer itself. so that there is no license issues I guess and these are only for the testing. I intentionally make them small enough, but I will move them another git repo if more data is required.